lib/actions.js

```diff
@@ -1,16 +1,18 @@
-import { bpanelClient } from '@bpanel/bpanel-utils';
-const client = bpanelClient();
+import { getClient } from '@bpanel/bpanel-utils';
+import { SET_PEERS } from './constants';
+
+const client = getClient();
```

Comment:
Should `getClient()` be invoked in the dispatch so that as clients change, an updated client will be used?

Reply:
Don’t think it’s necessary since this is using our Singleton mechanism. This just references the client Singleton and so should reflect the current state of the client whenever the dispatch is invoked.